.vsts.pipelines/jobs/ci-linux.yml

```diff
@@ -77,6 +77,23 @@ jobs:
     displayName: Build source-build
     timeoutInMinutes: 150
 
+  - script: |
+      set -ex
+      df -h
+      $(docker.run) $(docker.drop.map) $(docker.src.map) $(docker.src.work) $(imageName) /bin/bash -c "
+        mkdir -p /drop/artifacts/source-build
+        cp ./bin/*/*/*.tar.gz /drop/artifacts/source-build/
+        cp ./bin/*/*/runtime/*.tar.gz /drop/artifacts/source-build/
+        cp ./bin/obj/*/*/blobs/aspnetcore/Runtime/*/*.tar.gz /drop/artifacts/source-build/
+        cp ./bin/obj/*/*/blob-feed/packages/PackageVersions.props /drop/artifacts/source-build/
+        cp ./bin/obj/x64/Release/*.props /drop/artifacts/source-build/
+        cp -r ./bin/git-info /drop/artifacts/source-build/
+        cp -r ./bin/obj/*/*/build-info /drop/artifacts/source-build/"
+      du -h $(rootDirectory) | sort -h | tail -n 50
```

Comment:
I wonder if it would be better to have in the build itself... this seems like a useful diagnosis artifact someone can give us when source-build doesn't work for them.

The paths also might also be clearer in that context--using the project props rather than writing out full names in multiple places.